client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -564,6 +584,98 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     commitManager.handleGetReducerFileGroup(context, shuffleId)
   }
 
+  private def handleGetShuffleIdForApp(
```

Comment:
If we can add simple UTs for these changes, it will be great. It will encourage new contributors when they make changes to the same classes.

Reply:
sure, I was thinking about the same thing
currently, only spark integration tests are added, will check how LifecycleManager is tested now

Reply:
@waitinfuture can you help to provide a example for UT of LifecycleManager as reference?

Reply:
> @waitinfuture can you help to provide a example for UT of LifecycleManager as reference?

You can refer to `WithShuffleClientSuite`